frontend/app/work_packages/directives/inplace_editor/inplace-editor-display-pane-directive.js

```diff
@@ -92,10 +92,13 @@ module.exports = function(
         scope.displayPaneController.placeholder = I18n.t('js.label_click_to_enter_description');
       }
 
-      scope.$watch('fieldController.isEditing', function(isEditing) {
+      scope.$watch('fieldController.isEditing', function(isEditing, oldIsEditing) {
         if (!isEditing) {
           $timeout(function() {
-            element.find('.inplace-editing--trigger-link').focus();
+            if (oldIsEditing) {
```

Comment:
Sorry, I am dumb.

What am I seeing here? Why does this work?

1. What is the content of `isEditing`? When is it true and when is it false?
2. If I understand correctly, `oldIsEditing` is the value of `isEditing` before it changed... For a boolean this leads to the natural question: Isn't `isEditing == !oldIsEditing`? Apparently not, because then your change would do nothing...